### PR TITLE
Create the build config component

### PR DESCRIPTION
**Pull request by jhellar**
# Motivation

https://issues.jboss.org/browse/AEROGEAR-7709

### CODE REVIEW

File: ui/src/components/BuildConfig.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+class BuildConfig extends Component {
+  render() {
+    return (
+      <div className="build-config">
+        <ComponentLabel>
+          Build Config
+        </ComponentLabel>
+        <Row>
+          <Col sm={12} md={12} lg={6}>
```
Comment (sedroche) [inline ui/src/components/BuildConfig.js]: Do we need sm and md?

File: ui/src/components/BuildConfig.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { Row, Col } from 'patternfly-react';
+import ComponentLabel from './ComponentLabel';
+
+import '../style/BuildConfig.css';
+
+class BuildConfig extends Component {
+  render() {
+    return (
+      <div className="build-config">
+        <ComponentLabel>
```
Comment (sedroche) [inline ui/src/components/BuildConfig.js]: https://github.com/aerogear/mobile-client-service/pull/19#discussion_r205737032 Same. I don't mind which one we use just so we remember to sync them up

**Comment by StephenCoady** [comment]
@sedroche I don't mind what approach we take. the base view is almost ready to start tying everything together. did you think the approach we took with your branch worked well?

**Comment by pawelpaszki** [comment]
@sedroche I will create one more feature branch, if that is ok

**Comment by pawelpaszki** [comment]
@scoady pushed. here is the branch:
https://github.com/aerogear/mobile-client-service/tree/AEROGEAR_7713

**Comment by StephenCoady** [comment]
@jhellar I am going to close this PR as it was easier for me to just pull your commits into my branch. I also used @sedroche 's Component Label as I already had that set up locally.

**Pull request merged**